client/landing/stepper/declarative-flow/internals/steps-repository/trial-acknowledge/use-verify-email.tsx

```diff
@@ -11,13 +11,14 @@ import { successNotice, warningNotice } from 'calypso/state/notices/actions';
 export const useVerifyEmail = () => {
 	const dispatch = useDispatch();
 	const user = useSelector( getCurrentUser ) as UserData;
```

Comment:
I'm going to move this `as` because I feel this was the source of our problem in the first place. The types already show that `null`s are possible.

```suggestion
	const user = useSelector( getCurrentUser );
```